docs/_static/vdr/v1.yaml

```diff
@@ -272,6 +258,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/VerificationMethod'
+        assertionMethod:
+          description: List of KIDs that may sign JWTs, JWSs and VCs
+          type: array
+          items:
+            type: string
+        authentication:
+          description: List of KIDs that may alter DID documents that they control
+          type: array
+          items:
+            type: string
+        capabilityDelegation:
```

Comment:
We don't use this one, so why should it be added?

Reply:
Because it can be returned (as @jelmerterwal found out). Maybe not when using the high-level APIs of VDR/DIDMan, but you can just update a DID Document, adding a `capabilityDelegation` (for whatever reason, maybe a Bolt?).
You actually might be able to add whatever to the DID Document (not specified in the DID core spec), but I think we should at least try to be complete when it comes to the basic properties of a DID document?

Else, we should ditch this spec altogether and point to either libraries or schemas people can use to parse/create DID documents, imo.

Reply:
I'm trying to rule out strict parsers

Reply:
I agree: "be lenient about what you accept, and strict about what you return" (or something in those words).